### PR TITLE
Use constants to reduce duplication

### PR DESCRIPTION
**Pull request by stefanlogue**
This PR addresses a SonarCloud issue where some strings were duplicated multiple times. These strings are now constants

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -47,9 +47,15 @@ var (
 	AFS       *afero.Afero = &afero.Afero{Fs: FS}
 )
 
+const (
+	AS_GIT_EDITOR = "as-git-editor"
+	ERROR_STRING  = "Error: %s"
+	SHIFT_TAB     = "shift+tab"
```
Comment (ccoVeille) [inline main.go:53]: constants in Go are usually not in uppercase, and barely never in "screaming snake case"

File: main.go [hunk 2]
```diff
@@ -47,9 +47,15 @@ var (
 	AFS       *afero.Afero = &afero.Afero{Fs: FS}
 )
 
+const (
+	AS_GIT_EDITOR = "as-git-editor"
+	ERROR_STRING  = "Error: %s"
```
Comment (ccoVeille) [inline main.go:52]:
```suggestion
	ERROR_STRING  = "Error: %s"
```
This one make no sense to me

Unless I miss something you are using it when calling fail.

Please prefix error with "Errors: " in fail function.

Also, if you are only passing an error to fail, please consider updating fail signature to accept an error. This way you format the error with fmt.Errorf if you need some formating

Comment (stefanlogue) [reply main.go:52]: There are a couple of multiline error messages such as
```go
fail(
	"\n%s\n%s\n\n%s\n\n",
	color.RedString(fmt.Sprintf("It looks like the commit failed.\nError: %s", err)),
	color.YellowString("To run it again without going through meteor's wizard, simply run the following command (I've copied it to your clipboard!):"),
	color.BlueString(printableCommitCommand),
)
```
Where the arguments to `fail` are actually just strings. I would have to split `fail` into different functions for each specific case (passing errors, passing strings...) which would be fine, just not sure if the generic approach is better or worse?

File: main.go (hunk 1 above)
Comment (stefanlogue) [reply main.go:53]: I wasn't aware of this standard, thank you!

File: main.go (hunk 2 above)
Comment (ccoVeille) [reply main.go]: Indeed.

Comment (stefanlogue) [reply main.go]: I could have a function for generating the error messages though, and allow `fail` to just accept strings instead. I think this is a change that I'll do in a different PR though

Comment (ccoVeille) [reply main.go]: It sounds a good idea

**Pull request merged**